docs/ref/models/expressions.txt

```diff
@@ -113,6 +113,17 @@ Although ``reporter.stories_filed = F('stories_filed') + 1`` looks like a
 normal Python assignment of value to an instance attribute, in fact it's an SQL
 construct describing an operation on the database.
 
+For fields that are string-expressions (or ``django.contrib.postgres.fields.ArrayField``), you can use the standard slicing
+notation. The indices used in slicing are 0-based. However, step-argument is not supported in slicing::
```

Comment:
I'd suggest:

> However, step-argument is not supported in slicing.

changed to:

> The step-argument is not supported. For example::

Also a `..versionadded` annotation is needed

Reply:
Changed this to your suggestion. Added a versionadded, although I'm not sure which line to put it on?